### DISCUSSION TITLE
Possibility of notification of WAN failover?

### DISCUSSION THREAD

**Discussion by m477r1x** [gnuton/asuswrt-merlin.ng#724] [Ideas]
It would be handy for me if I could receive an email or something, when the WAN fails from primary to secondary. Not sure how feasible this is to implement but would be handy for me personally for sure. Maybe others can chime in if it's something they are interested in.

**Comment by decoderman** [comment]
Dual WAN failover is available through amtm and can send mails: https://www.snbforums.com/threads/dual-wan-failover-v2-release.83674/
